locomotion-sdk/src/pages/AuthScreens/Onboarding/Name.js

```diff
@@ -31,15 +31,26 @@ const Name = () => {
         />
         <TextInput
```

Comment:
I think it's better to create a base/generic input field and manage the focused style there, we will use it all across the app